test/AutoDiff/activity_analysis.swift

```diff
@@ -121,91 +187,101 @@ func testArrayUninitializedIntrinsicAddress(_ x: Float, _ y: Float) -> [Float] {
-    if true {
-      result = result * x
-    }
-  }
-  return result
-}
-
-// CHECK-LABEL: [AD] Activity info for ${{.*}}TF_781{{.*}} at (source=0 parameters=(0))
 // CHECK: [ACTIVE] %0 = argument of bb0 : $Float
-// CHECK: [USEFUL] %1 = argument of bb0 : $Float
-// CHECK: [ACTIVE]   %4 = alloc_stack $Float, var, name "result"
-// CHECK: [ACTIVE]   %19 = begin_access [read] [static] %4 : $*Float
-// CHECK: [ACTIVE]   %20 = load [trivial] %19 : $*Float
-// CHECK: [ACTIVE]   %23 = apply %22(%20, %0, %18) : $@convention(method) (Float, Float, @thin Float.Type) -> Float
-// CHECK: [ACTIVE]   %24 = begin_access [modify] [static] %4 : $*Float
-// CHECK: [ACTIVE]   %31 = begin_access [read] [static] %4 : $*Float
-// CHECK: [ACTIVE]   %32 = load [trivial] %31 : $*Float
-
-// TF-954: check nested conditionals and addresses.
+// CHECK: [ACTIVE] %1 = argument of bb0 : $Float
+// CHECK: [USEFUL]   %4 = integer_literal $Builtin.Word, 2
+// CHECK: [NONE]   // function_ref _allocateUninitializedArray<A>(_:)
+// CHECK: [ACTIVE]   %6 = apply %5<Float>(%4) : $@convention(thin) <τ_0_0> (Builtin.Word) -> (@owned Array<τ_0_0>, Builtin.RawPointer)
+// CHECK: [ACTIVE] (**%7**, %8) = destructure_tuple %6 : $(Array<Float>, Builtin.RawPointer)
+// CHECK: [VARIED] (%7, **%8**) = destructure_tuple %6 : $(Array<Float>, Builtin.RawPointer)
+// CHECK: [ACTIVE]   %9 = pointer_to_address %8 : $Builtin.RawPointer to [strict] $*Float
+// CHECK: [NONE]   // function_ref static Float.* infix(_:_:)
+// CHECK: [ACTIVE]   %12 = apply %11(%0, %1, %10) : $@convention(method) (Float, Float, @thin Float.Type) -> Float
+// CHECK: [VARIED]   %14 = integer_literal $Builtin.Word, 1
+// CHECK: [ACTIVE]   %15 = index_addr %9 : $*Float, %14 : $Builtin.Word
+// CHECK: [USEFUL]   %16 = metatype $@thin Float.Type
+// CHECK: [NONE]   // function_ref static Float.* infix(_:_:)
+// CHECK: [ACTIVE]   %18 = apply %17(%0, %1, %16) : $@convention(method) (Float, Float, @thin Float.Type) -> Float
 
+// TF-975: Test nested array literals.
+// FIXME(TF-975): Some values are incorrectly not marked as active: `%0`, `%1`, etc.
 @differentiable
-func TF_954(_ x: Float) -> Float {
-  var outer = x
-  outerIf: if true {
-    var inner = outer
-    inner = inner * x // check activity of this `apply`
-    if false {
-      break outerIf
-    }
-    outer = inner
-  }
-  return outer
+func testArrayUninitializedIntrinsicNested(_ x: Float, _ y: Float) -> [Float] {
+  let array = [x, y]
+  return [array[0], array[1]]
 }
+// CHECK-LABEL: [AD] Activity info for ${{.*}}testArrayUninitializedIntrinsicNested{{.*}} at (source=0 parameters=(0 1))
+// CHECK: [VARIED] %0 = argument of bb0 : $Float
+// CHECK: [VARIED] %1 = argument of bb0 : $Float
+// CHECK: [NONE]   %4 = integer_literal $Builtin.Word, 2
+// CHECK: [NONE]   // function_ref _allocateUninitializedArray<A>(_:)
+// CHECK: [VARIED]   %6 = apply %5<Float>(%4) : $@convention(thin) <τ_0_0> (Builtin.Word) -> (@owned Array<τ_0_0>, Builtin.RawPointer) // user: %7
```

Comment:
Oops, `user:` comments should be removed.

```suggestion
// CHECK: [VARIED]   %6 = apply %5<Float>(%4) : $@convention(thin) <τ_0_0> (Builtin.Word) -> (@owned Array<τ_0_0>, Builtin.RawPointer)
```